hooks/mari/publish_textures.py

```diff
@@ -93,6 +93,8 @@ def validate(self, task_settings, item):
         :param item: Item to process
         :returns: True if item is valid, False otherwise.
         """
+        # textures should always contain UDIM or tag <UDIM>
+        item.properties["is_sequence"] = True
```

Comment:
This should be set in the collector, not here

Reply:
Good point. Changing.